README.md

```diff
@@ -119,6 +124,19 @@ Returned values
 # Returns `nil` in case does not match any type.
 ```
 
+## Convert from one format to another
+```ruby
+require 'subtitle'
+
+subtitle = Subtitle.new(caption_file_path)
+subtitle.transform(types_to_convert, src_lang, target_lang, options)
```

Comment:
Let's align it similar to translate doc; providing different options and its significance.

the order can be changed to "types_to_covert", options, target_lang & src_lang; so we can void explicit nil passing. what do you think?.

Reply:
Will do !!